exampleCourse/questions/elementOverlayFigure/question.html

```diff
@@ -0,0 +1,20 @@
+<pl-question-panel>
+    <p>
+        The element <code class="user-output">pl-overlay</code> allows the positioning of pre-existing elements in arbitrary positions on top of other elements.
+        This question combines the <code class="user-output">pl-figure</code> element to serve as a background,
+        along with the <code class="user-output">pl-string-input</code> element.
+    </p>
+    <p>
+        Write the name of the missing country:
+    </p>
+</pl-question-panel>
+
+<pl-overlay width="400" height="600" clip="false">
+    <pl-location x="0" y="0">
+       <pl-figure file-name="mapzoom.png" directory="clientFilesQuestion"></pl-figure>
```

Comment:
Mmm, looking at the raw of the image, Ukraine is already labeled. How does this impact rendering of the problem? (e.g. a student right clicks to open the image in a new browser tab?)

Reply:
I didn't think of this. I was just trying to find something simple and easy to do. In a real example, the figure will have the missing information. We can edit the figure, if that is important?